app/src/release/java/org/fossasia/susi/ai/rest/ClientBuilder.java

```diff
@@ -0,0 +1,50 @@
+package org.fossasia.susi.ai.rest;
+
+import org.fossasia.susi.ai.helper.PrefManager;
+import org.fossasia.susi.ai.rest.interceptors.TokenInterceptor;
+
+import okhttp3.OkHttpClient;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+
+/**
+ * Created by saurabh on 1/10/16.
+ * <p>
+ * Singleton class to get Susi client.
+ */
+
+public class ClientBuilder {
```

Comment:
I think this can be done by using just checking debug mode in ClientBuilder.
```java
if (BuildConfig.DEBUG) {
  // do something for a debug build
}```

do not need to make a new file to increase PRs